src/View/UIViewTrait.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace BootstrapUI\View;
+
+/**
+ * UIViewTrait: Trait that aims to:
+ *
+ * loading the custom UIBootstrap helpers.
+ * setting the View layout to the UIBootstrap's one.
+ */
+trait UIViewTrait
+{
+    /**
+     * Initialization hook method.
+     *
+     * @return void
+     */
+    public function initializeUI()
+    {
+        //Set the layout to BootstrapUI's layout
+        $this->layout = 'BootstrapUI.default';
```

Comment:
The layout should be set only if current value is `default`, otherwise it will override custom layout passed through controller which is set in View::__construct() before initialize() is called.